Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Queries/Person/GetNearestDrivers.hs

```diff
@@ -32,7 +34,7 @@ data NearestDriversResult = NearestDriversResult
   deriving (Generic, Show, PrettyShow, HasCoordinates)
 
 getNearestDrivers ::
-  (MonadFlow m, MonadTime m) =>
+  (MonadFlow m, MonadTime m, MonadReader r m, HasField "enableLocationTrackingService" r Bool, HasFlowEnv m r '["ltsCfg" ::: LT.LocationTrackingeServiceConfig], CoreMetrics m) =>
```

Comment:
Why `MonadReader`? It's already there in MonadFlow.

Reply:
removed